cobalt/build/configs/linux_common.gn

```diff
@@ -87,11 +86,6 @@ use_pangocairo = false
 # We are not using tflite in cobalt currently.
 build_with_tflite_lib = false
 
-# Overriding from third_party/blink/renderer/modules/ml/webnn/features.gni:10
-# webnn is a machine learning related module: https://chromestatus.com/feature/5176273954144256
-# We are not using webnn in cobalt currently.
-build_webnn_with_xnnpack = false
```

Comment:
Same q : as above - is this intended to be temporary removal ? Or upstream doesn't have it anymore at all ?